var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -16,9 +16,8 @@ class Zfp(MakefilePackage):
     url      = 'http://computing.llnl.gov/projects/floating-point-compression/download/zfp-0.5.2.tar.gz'
 
     version('0.5.4', sha256='768a05ed9bf10e54ac306f90b81dd17b0e7b13782f01823d7da4394fd2da8adb')
-    version('0.5.2', '2f0a77aa34087219a6e10b8b7d031e77')
-    version('0.5.1', '0ed7059a9b480635e0dd33745e213d17')
-    version('0.5.0', '2ab29a852e65ad85aae38925c5003654')
+    version('0.5.2', sha256='89e718edb966422b5898b5c37b1b0a781d4effacb511520558469e3ff7f65d7c')
```

Comment:
Can we add a line for 0.5.5? It is now available. Also, the `url =` line above, line ~16, I think that gives the default right. I think that should point to 0.5.5 now.

Reply:
I see the new line 18 for 0.5.5. Thanks! 

What about the version referenced in line 16? Does that matter at all here or is it just used as a pattern to inform spack how to construct other version urls?

Reply:
> is it just used as a pattern to inform spack how to construct other version urls?

Yep